src/heitech.zer0mqXt.core/Patterns/PubSub.cs

```diff
@@ -20,21 +20,41 @@ public PubSub(SocketConfiguration configuration)
             _configuration = configuration;
         }
 
+        private PublisherSocket _publisherSocket;
+
+        internal void PrimePublisher()
+        {
+            try
+            {
+                _publisherSocket = new PublisherSocket();
+                _publisherSocket.Bind(_configuration.Address());
+            }
+            catch (System.Exception ex)
```

Comment:
We already imported the `System` namespace in this file - we could simplify `Exception` here.